media-gfx/freecad/freecad-0.16.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils fortran-2 python-single-r1 fdo-mime
+
+DESCRIPTION="QT based Computer Aided Design application"
+HOMEPAGE="http://www.freecadweb.org/"
+SRC_URI="https://github.com/FreeCAD/FreeCAD/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+#sci-libs/orocos_kdl waiting for Bug 604130 (keyword ~x86)
+#dev-qt/qtgui:4[-egl] and dev-qt/qtopengl:4[-egl] : Bug 564978 
+COMMON_DEPEND="dev-cpp/eigen:3
+	dev-libs/boost[python]
+	dev-libs/xerces-c[icu]
+	dev-python/matplotlib
+	dev-python/pyside[X]
+	dev-python/shiboken
+	dev-qt/designer:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
+	dev-qt/qtsvg:4
+	dev-qt/qtwebkit:4
+	media-libs/coin
+	|| ( sci-libs/opencascade:6.9.1[vtk] sci-libs/opencascade:6.9.0[vtk] sci-libs/opencascade:6.8.0 sci-libs/opencascade:6.7.1 )
+	sys-libs/zlib
+	virtual/glu
+	media-libs/freetype
+	dev-java/xerces
+	${PYTHON_DEPS}"
+RDEPEND="${COMMON_DEPEND}
+	dev-qt/assistant:4
+	dev-python/pivy
+	dev-python/numpy"
+DEPEND="${COMMON_DEPEND}
+	dev-python/pyside-tools
+	>=dev-lang/swig-2.0.4-r1:0"
+
```

Comment:
you're missing `REQUIRED_USE="${PYTHON_REQUIRED_USE}"` which is required for `python*-r1.eclass`